torchvision/models/detection/faster_rcnn.py

```diff
@@ -30,21 +30,21 @@ class FasterRCNN(GeneralizedRCNN):
 
     The behavior of the model changes depending if it is in training or evaluation mode.
 
-    During training, the model expects both the input tensors, as well as a targets dictionary,
+    During training, the model expects both the input tensors, as well as a targets (list of dictionary),
     containing:
-        - boxes (Tensor[N, 4]): the ground-truth boxes in [x0, y0, x1, y1] format, with values
+        - boxes (FloatTensor[N, 4]): the ground-truth boxes in [x0, y0, x1, y1] format, with values
```

Comment:
Those are great changes, thanks!

But I'm a bit concerned that this looks like the legacy interface of `torch.FloatTensor`, etc.
I wonder if there is a better way of representing this? For example, in numpy, everything is a `ndarray`, but with different types.

Thoughts?

Reply:
Would the descriptions like `boxes(Tensor[N, 4], dtype=torch.float)` or `boxes(FloatTensor(N, 4))` be better? 

`(Tensor[N, 4], dtype=torch.float)` is not a valid syntax but conveys the requirement whereas `FloatTensor(N, 4)` would actually create a float tensor with `dim = (N, 4)`.

Do let me know if you have any better way of representing them?

Reply:
Let's just keep this as is for now. But I'd like to remove the `FloatTensor`, changes, and only keep `Int64Tensor` instead, because boxes is not required to be `Float`